DocumentedExamples/regional-mom6-forced-by-access-om2.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<p>why there are 3 commented out lines of code @luweiyang?</p><p>if we need them, then uncomment,</p><p>if we don't need them then delete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I don't think we need them. L20 and L21 are the same as L6 and L7 for selecting data along the dims <code>yu_ocean</code> and <code>yt_ocean</code>. I will delete them.</p>

Reply:
Generally, commented code is bad...
Either we want it or we delete it. 
It's good to let go. :)